std/net/curl.d

```diff
@@ -2314,8 +2314,18 @@ private bool decodeLineInto(Terminator, Char = char)(ref const(ubyte)[] basesrc,
   *     (in char[] key, in char[] value) { writeln(key ~ ": " ~ value); };
   * http.onReceive = (ubyte[] data) { /+ drop +/ return data.length; };
   * http.perform();
+  * ---
+  *
+  */
+
+/**
+  * HTTP Put with data senders
```

Comment:
Punctuation, capitalization. `HTTP` is redundant. Perhaps: `Put with data senders:`